source/app/models/Game_Brain.rb

```diff
@@ -0,0 +1,26 @@
+class GameBrain
+  def initialize(game)
+    @player_one = game.player_one_id
+    @player_two = game.player_two_id
+    @player_one_choice = game.choice_one_id
+    @player_two_choice = game.choice_two_id
+  end
+
+  def winner
+    if (@player_one_choice == 1 && @player_two_choice == 2)
```

Comment:
I'd recommend passing in strings rather than the token IDs, and using a hash instead of a big if statement. It'll make the code much easier to read.

  BEATS = {'rock' => 'scissors', 'scissors' => 'paper', 'paper' => 'rock' }
  def winner
    return "tie" if @player_one_choice == @player_two_choice
    return @player_one if BEATS[@player_one_choice] == @player_two_choice
    @player_two
  end